I/ITK/build_tarballs.jl

```diff
@@ -37,140 +46,213 @@ cmake -B build -S . \
     -DVXL_HAS_SSE2_HARDWARE_SUPPORT:STRING=1 \
     -DVCL_HAS_LFS:STRING=1 \
     -DDOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS:STRING=1 \
-    -DHAVE_CLOCK_GETTIME_RUN:STRING=0
-cmake --build build --parallel ${nproc}
+    -DHAVE_CLOCK_GETTIME_RUN:STRING=0 \
+    -D_libcxx_run_result:STRING=0 \
+    -D_libcxx_run_result__TRYRUN_OUTPUT:STRING=0 \
+    -DCMAKE_CXX_STANDARD=14 \
+    -DCMAKE_CXX_STANDARD_REQUIRED=ON \
+    -DITK_LEGACY_REMOVE=OFF \
+    -DITK_BUILD_TESTING=OFF \
+    -DBUILD_TESTING=OFF \
+    -DITK_USE_WIN32_LIBS=ON \
+    -DITK_SKIP_PATH_LENGTH_CHECKS=ON \
+    -DITK_SKIP_PATH_LENGTH_CHECKS=1
+
+cmake --build build --parallel ${nproc} -- -k
 cmake --install build
 install_license ${WORKSPACE}/srcdir/ITK/LICENSE
+
 if [[ "${target}" == *x86_64-w64-mingw32* ]]; then
-    cp $prefix/lib/libitkminc2-5.3.dll $prefix/bin
-    cp $prefix/lib/libitkminc2-5.3.dll.a $prefix/bin
+    # Ensure all DLLs are in the correct location
+    mkdir -p ${prefix}/bin
+    cp $prefix/lib/*.dll ${prefix}/bin/ || true
```

Comment:
Please don't use `|| true`, just make it work properly. Also, use verbose versions of commands to make it possible to see in log what happened